### PR TITLE
Add Atlantic 2024 full site and tickets

### PR DESCRIPTION
**Pull request by mxsasha**
- [x] Open on tito
- [x] Update grant link
- [x] Finalise ticket pricing

----
📚 Documentation preview 📚: https://writethedocs-www--2120.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/atlantic/2024/news/tickets-grants.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+-----------------------------------
+
+Every year we are fortunate to have the support of like-minded organizations and communities, who believe in the collaborative and welcoming community spirit and help us make our events great.
+
+We have published our `virtual sponsorship prospectus`_ for the conference,
+and are open for new sponsorship inquiries.
+
+.. _virtual sponsorship prospectus: https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/
+
+We will still run the `job fair <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/job-fair/>`_, helping documentarians and organizations connect with each other. Check out the sponsorship prospectus for more details and to reserve your spot today.
```
Comment (ericholscher) [inline docs/conf/atlantic/2024/news/tickets-grants.rst]: I wonder about leaving this out, since we haven't had luck filling it the last couple years?

**Review by ericholscher — APPROVED** [review]
Announcement looks good, and 👍 on the removal of Hopin at this point.

File: docs/conf/atlantic/2024/news/tickets-grants.rst (hunk 1 above)
Comment (mxsasha) [reply docs/conf/atlantic/2024/news/tickets-grants.rst]: I'll turn it into something more generic.

File: docs/conf/atlantic/2024/unconference.rst [hunk 2]
```diff
@@ -33,29 +33,29 @@ To lead an unconf session, propose a topic and pick a time; unconf sessions are
 
 {% if unconf and unconf.url %}
 
-Sessions take place in Hopin, but you propose a session by adding your topic to a timeslot and table number in the  `Unconference sign-up sheet <{{unconf.url}}>`__.
+Sessions take place in the virtual conference platform, but you propose a session by adding your topic to a timeslot and table number in the  `Unconference sign-up sheet <{{unconf.url}}>`__.
```
Comment (plaindocs) [inline docs/conf/atlantic/2024/unconference.rst:36]: Looks good. We should probably add a virtual conf platform variable?

**Review by plaindocs — APPROVED** [review]
:+1: 

> Interested in being a speaker? We’re accepting [Call for Proposals](http://127.0.0.1:8888/conf/atlantic/2024/cfp/) until May 15, 2023.

Probably mixed by merging Eric's dates PR.

**Comment by plaindocs** [comment]
Also, `at2023-tickets` :japanese_ogre:

File: docs/conf/atlantic/2024/unconference.rst (hunk 2 above)
Comment (mxsasha) [reply docs/conf/atlantic/2024/unconference.rst:36]: There aren't that many places, except the welcome wagon guide, which will need a rewrite anyways

**Pull request merged**